processes/external-contributions/README.MD

```diff
@@ -0,0 +1,90 @@
+# How to Handle External Contributions
+
+## Overview
+
+- Decide whether the PR is adding community features or affects "supported" functionality.
```

Comment:
I would alter the dichotomy to be "Community Supported" and "Officially Supported" - the community does indeed support their features